RULES.md

```diff
@@ -262,6 +264,15 @@ Empty csv file found in the archive: file does not have any headers, or is a req
 ##### References:
 * [GTFS files requirements](https://gtfs.org/reference/static#file-requirements)
 
+#### EqualShapeDistanceNotice
+
+<a name="EqualShapeDistanceNotice"/>
+
+When sorted by `shape.shape_pt_sequence`, two consecutive shape points must not have equal values for `shape_dist_traveled` if their GPS coordinates are the same.
```

Comment:
I think this should read "if their GPS coordinates are not the same". It's an error to have equal distances with different coordinates, it's a warning to have equal distances and equal coordinates.

Reply:
My only other concern is that at least for our feed, which generates equal distances every stop pair, we get 10s of thousands of these notices per validation. I'd love to have the ability to output an abbreviated json validation report that reported the number of times each notices happened, but only the first X occurrences.

Reply:
> I think this should read "if their GPS coordinates are not the same". 

👍🏾
> It's an error to have equal distances with different coordinates, it's a warning to have equal distances and equal coordinates.

Or if said coordinates are actually less than 1 meter away according to https://github.com/MobilityData/gtfs-validator/pull/1083#discussion_r774106569. 

> My only other concern is that at least for our feed, which generates equal distances every stop pair, we get 10s of thousands of these notices per validation. I'd love to have the ability to output an abbreviated json validation report that reported the number of times each notices happened, but only the first X occurrences.

As of now a maximum of 1000 of notices are exported by type and severity, and the validation report gives information about the total number of notices per type and severity. 

https://github.com/MobilityData/gtfs-validator/blob/a871d4bad3f7eb7b2d17d1122fd825aca6883b86/core/src/main/java/org/mobilitydata/gtfsvalidator/notice/NoticeContainer.java#L54

https://github.com/MobilityData/gtfs-validator/blob/a871d4bad3f7eb7b2d17d1122fd825aca6883b86/core/src/main/java/org/mobilitydata/gtfsvalidator/notice/NoticeContainer.java#L190-L193

> I'd love to have the ability to output an abbreviated json validation report that reported the number of times each notices happened, but only the first X occurrences.

I see this as separate from the bug fixed here. We could allow users to reduce this limit  in a separate PR by making the limit user configurable. cc @isabelle-dr.